docs/team/Kzrthikz.adoc

```diff
@@ -0,0 +1,88 @@
+= Murugesan Karthika - Project Portfolio
+ :site-section: AboutUs
+ :imagesDir: ../images
```

Comment:
![image](https://user-images.githubusercontent.com/43642522/68543673-65eb1f00-03f5-11ea-8b4a-a05bc5ba5ac2.png)

This section did not get parsed properly. Remove the leading whitespaces